src/common/Utilities.py

```diff
@@ -67,7 +67,7 @@ def existing_filepath(data_path, file_name):
     data_path: path to the file
     file_name: file name
     '''
-    full_name = data_path + '/' + file_name
+    full_name = os.path.join(os.path.abspath(data_path), file_name)
```

Comment:
nice one. This might explain why we currently need on Windows
```
 # need to be forward slashes 
 $Env:SIRF_PATH = "C:/Users/krisf/Documents/devel/buildVC/SIRF-SuperBuild/sources/SIRF"
```
https://github.com/SyneRBI/SIRF-SuperBuild/issues/460

@evgueni-ovtchinnikov after this has merged, could you try to set `SIRF_PATH` with backslashes and see if it works?